sdk/servicebus/azure-messaging-servicebus/TROUBLESHOOTING.md

```diff
@@ -0,0 +1,153 @@
+# Troubleshooting Service Bus issues
+
+This troubleshooting guide covers failure investigation techniques, common errors for the credential types in the Azure Service Bus Java client library, and mitigation steps to resolve these errors.
+
+## Table of contents
+- [Implicit prefetch issue in ServiceBusReceiverClient](#implicit-prefetch-issue-in-servicebusreceiverclient)
+- [Troubleshoot ServiceBusProcessorClient issues](#troubleshoot-servicebusprocessorclient-issues)
+  - [Client hangs or stalls with a high prefetch and maxConcurrentCall value](#client-hangs-or-stalls-with-a-high-prefetch-and-maxconcurrentcall-value)
+    - [Credit calculation issue](#credit-calculation-issue)
+- [Autocomplete issue](#autocomplete-issue)
+- [Migrate from legacy to new client library](#migrate-from-legacy-to-new-client-library)
+- [Enable and configure logging](#enable-and-configure-logging)
+    - [Configuring Log4J 2](#configuring-log4j-2)
+    - [Configuring logback](#configuring-logback)
+    - [Enable AMQP transport logging](#enable-amqp-transport-logging)
+    - [Reduce logging](#reduce-logging)
+- [Get additional help](#get-additional-help)
+    - [Filing GitHub issues](#filing-github-issues)
+
```

Comment:
Event Hubs troubleshooting guide mention about AMQP exceptions: https://github.com/Azure/azure-sdk-for-java/blob/main/sdk/eventhubs/azure-messaging-eventhubs/TROUBLESHOOTING.md#handle-event-hubs-exceptions

Do we need to copy the content to service bus or add a link? It is common that user could see `amqp:connection:forced` or `amqp:link:detach-forced`  on their end and ask why.

Reply:
For exception handling, we need to rewrite some of the ways in which we throw. This is why I left out the exception handling section for now. Let's add it after the exception handling change.